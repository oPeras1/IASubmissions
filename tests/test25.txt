1	1	1	1	1	1	1	1	2	3	4	5	5	5	5	6	6	6	7	7	8	8	8	9	9	10	10	10	10	11
12	12	13	13	13	13	13	2	2	3	4	5	14	14	5	6	15	7	7	7	8	16	8	9	9	9	10	10	11	11
12	13	13	17	17	17	17	2	2	3	4	5	18	14	5	15	15	15	15	7	7	16	8	9	9	9	19	19	11	20
12	12	12	21	17	2	2	2	22	3	4	5	18	14	5	23	23	23	23	7	24	16	8	25	25	26	26	19	20	20
12	12	21	21	27	27	27	22	22	3	3	5	18	18	23	23	23	23	7	7	24	16	16	16	25	25	26	19	20	20
28	28	21	27	27	27	3	3	22	3	5	5	5	18	18	29	23	30	30	24	24	24	24	16	16	26	26	31	31	20
28	32	27	27	32	3	3	3	3	3	3	18	18	18	29	29	29	30	30	33	24	34	24	24	34	26	31	31	20	20
28	32	32	32	32	3	3	35	35	3	36	37	18	37	29	29	29	30	30	33	34	34	34	24	34	26	31	31	38	34
39	39	39	39	40	40	3	3	35	3	36	37	37	37	41	41	29	41	41	33	33	42	34	34	34	26	38	38	38	34
43	39	43	40	40	40	40	44	35	36	36	36	37	45	41	41	41	41	46	33	42	42	42	34	34	34	34	34	34	34
43	43	43	40	47	40	44	44	35	48	49	49	37	45	50	50	50	41	46	46	51	51	51	51	51	52	52	52	34	53
43	54	43	47	47	55	44	56	48	48	49	45	45	45	50	57	58	58	46	46	51	59	59	59	51	51	52	60	53	53
54	54	54	54	47	55	44	56	61	48	49	45	45	62	57	57	58	58	58	46	51	59	63	63	51	51	60	60	64	53
54	65	65	54	55	55	56	56	61	66	49	45	62	62	57	57	67	67	68	51	51	63	63	63	63	51	60	64	64	64
54	69	65	54	55	56	56	61	61	66	49	49	70	62	57	57	67	67	68	68	51	63	51	51	51	60	60	71	64	64
69	69	65	54	55	55	55	61	61	66	49	70	70	72	57	67	67	67	68	51	51	51	51	51	51	51	71	71	71	64
69	73	65	74	55	61	61	61	75	66	49	49	70	72	72	76	67	67	77	77	51	78	78	78	79	80	81	81	71	64
69	73	73	74	74	61	82	75	75	49	49	70	70	72	72	76	76	83	83	77	51	78	84	78	79	80	80	81	81	64
69	73	73	74	74	61	82	75	75	75	49	70	70	76	76	76	76	83	83	77	77	78	84	79	79	80	80	80	81	64
85	73	74	74	74	82	82	86	86	87	87	70	87	83	83	83	83	83	83	88	84	84	84	89	79	90	91	91	81	91
85	85	74	74	92	82	86	86	86	86	87	87	87	87	87	87	87	87	87	88	88	84	90	89	89	90	91	91	91	91
85	85	93	93	92	94	94	86	95	95	96	96	97	97	97	97	97	97	87	98	88	88	90	89	90	90	90	99	99	99
85	93	93	92	92	94	94	100	100	95	96	101	97	97	101	97	102	87	87	98	88	88	90	89	89	90	99	99	99	103
93	93	92	92	92	94	104	104	100	95	96	101	101	101	101	102	102	102	98	98	98	105	90	90	90	90	99	103	103	103
93	106	92	92	104	104	104	100	100	95	96	96	101	107	108	108	108	108	108	108	105	105	105	105	90	90	103	103	103	109
106	106	92	92	104	110	110	100	100	95	95	96	107	107	108	111	111	108	111	112	112	105	105	105	113	113	109	109	109	109
106	106	92	92	114	110	110	110	100	95	107	107	107	115	108	108	111	111	111	116	112	112	112	113	113	113	113	113	113	113
117	117	92	92	114	114	110	110	100	95	95	118	107	115	115	119	111	120	116	116	112	121	112	121	122	122	122	122	113	113
114	117	117	114	114	123	123	110	124	118	118	118	107	115	115	119	120	120	116	112	112	121	121	121	121	121	121	122	125	113
114	114	114	114	123	123	123	124	124	124	118	107	107	115	119	119	120	120	120	120	112	112	121	126	126	126	126	125	125	125