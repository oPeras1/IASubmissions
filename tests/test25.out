1	1	1	I	I	I	I	1	S	3	I	5	5	5	5	L	L	L	7	7	L	L	L	T	9	T	T	T	10	S
I	12	S	S	13	13	13	S	S	3	I	5	L	L	5	L	15	7	7	7	L	16	8	T	T	9	T	10	S	S
I	S	S	17	L	L	L	S	2	3	I	5	18	L	5	I	I	I	I	L	7	L	8	T	9	9	L	L	S	20
I	12	12	S	L	2	2	2	T	3	I	L	18	L	5	S	23	23	23	L	24	L	8	S	S	26	26	L	20	I
I	12	S	S	27	S	S	T	T	3	3	L	S	S	23	S	S	23	L	L	24	L	L	16	S	S	26	L	20	I
L	L	S	27	S	S	3	3	T	3	L	L	5	S	S	29	S	T	30	24	24	24	S	16	16	I	26	S	S	I
L	32	27	27	L	3	T	3	3	3	3	18	18	18	L	29	29	T	T	I	24	34	S	S	34	I	S	S	20	I
L	32	L	L	L	T	T	L	L	3	L	37	18	S	L	L	L	T	30	I	34	34	34	S	34	I	31	31	L	34
T	T	T	39	40	40	T	3	L	3	L	37	S	S	41	41	29	S	S	I	33	T	34	L	34	I	L	L	L	34
43	T	43	I	I	I	I	44	L	36	L	L	S	45	41	41	S	S	46	I	T	T	T	L	L	L	34	34	34	34
L	L	L	40	T	40	L	L	35	T	49	49	37	45	L	L	L	41	L	L	51	51	51	51	51	T	T	T	34	T
43	54	L	T	T	55	L	56	T	T	49	L	L	45	L	57	S	S	46	L	51	L	L	L	51	51	T	60	T	T
L	L	54	54	T	55	L	56	61	T	49	L	45	T	57	L	58	S	S	L	51	L	63	63	51	51	L	L	64	T
L	65	I	54	I	55	S	S	61	I	49	L	T	T	57	L	67	67	T	51	51	I	I	I	I	51	L	64	L	L
L	69	I	54	I	S	S	61	61	I	49	49	70	T	L	L	67	67	T	T	51	63	51	51	T	60	L	S	64	L
I	69	I	54	I	55	55	S	S	I	49	70	70	S	57	S	S	67	T	51	51	51	51	T	T	T	71	S	S	L
I	73	I	74	I	61	S	S	75	I	L	L	70	S	S	76	S	S	L	L	51	L	L	L	79	S	81	81	S	64
I	73	S	74	T	61	L	75	T	49	L	70	S	72	S	76	76	T	83	L	51	L	84	78	T	S	S	L	L	64
I	S	S	T	T	61	L	T	T	T	L	S	S	I	I	I	I	T	T	L	77	78	L	T	T	80	S	80	L	64
85	S	74	74	T	L	L	86	86	87	87	S	87	83	83	83	83	T	83	88	L	L	L	89	T	90	91	91	L	91
L	L	74	74	L	82	I	I	I	I	87	87	87	87	87	I	I	I	I	88	S	84	90	L	L	90	I	I	I	I
L	85	S	93	L	94	S	86	95	95	L	L	97	L	L	L	97	97	87	L	S	S	90	L	90	90	90	S	99	99
L	S	S	L	L	S	S	L	L	95	L	101	97	L	101	97	T	87	87	L	88	S	90	L	89	90	S	S	99	S
93	S	92	92	92	S	104	104	L	95	L	T	T	T	101	T	T	T	L	L	98	T	90	T	T	T	S	103	S	S
93	T	92	92	L	L	L	100	L	I	96	96	T	107	L	L	108	108	108	108	T	T	T	105	T	90	103	103	S	109
T	T	92	92	L	110	S	100	100	I	95	96	L	107	L	111	T	108	111	112	L	105	105	105	S	S	I	I	I	I
106	T	92	92	T	110	S	S	100	I	L	L	L	I	L	108	T	T	111	S	L	L	L	S	S	113	113	113	113	113
S	S	92	92	T	T	110	S	100	I	95	S	107	I	115	L	T	120	S	S	112	S	112	121	I	I	I	I	113	113
114	S	S	114	T	123	L	110	T	118	S	S	107	I	115	L	120	120	S	112	112	S	S	121	121	121	121	122	T	113
114	114	114	114	L	L	L	T	T	T	S	107	107	I	L	L	I	I	I	I	112	112	S	I	I	I	I	T	T	T